include/maliput/routing/route.h

```diff
@@ -233,6 +232,25 @@ class Route final {
   /// @throws common::assertion_error When @p start_position is not valid.
   api::LaneSRoute ComputeLaneSRoute(const api::RoadPosition& start_position) const;
 
+  /// Evaluates whether this Route is connected end to end.
+  ///
+  /// End to end connectivity for a Route implies the following conditions:
+  /// - The first Phase contains only one start position.
+  /// - The last Phase contains only one end position.
+  /// - The number of end positions in a given Phase and start positions in the following Phase are the same.
+  /// - For each position in the end positions in a given Phase there is a one-to-one relation in the start
+  ///   positions in the followin Phase such that:
+  ///   - The positions are coincident in the INERTIAL-Frame (geometrical connectivity) and,
+  ///   - The positions belong to the same api::Lane or are endpoints of api::Lanes that share
+  ///     oppossing api::LaneEnds in an api::Branchpoint (tological connectivity).
+  ///
+  /// Router implementations are expected to construct Routes and validate they are connected end to end.
+  /// Users of a Route must expect Routes built by Router implementations to be end to end connected.
```

Comment:
Should "must expect" be "may expect"?

"must" seems overly restrictive.

Reply:
[5f99c1c](https://github.com/maliput/maliput/pull/628/commits/5f99c1c613ff4ae3206bf0796b0287698f0d08ae)